### PR TITLE
GUB-42502: Updated sdk to connect using TLS 1.2

### PR DESCRIPTION
**Pull request by mikephillipsjr**

### CODE REVIEW

**Review by smedrick — APPROVED** [review]

File: Granicus.MediaManager.UserSDK/MediaManager.cs [hunk 1]
```diff
@@ -61,6 +61,7 @@ public class MediaManager : SoapHttpClientProtocol, IMediaManager
         /// </remarks>
         public MediaManager()
         {
+            System.Net.ServicePointManager.SecurityProtocol = SecurityProtocolTypeExtensions.Tls12;
```
Comment (umidt) [inline Granicus.MediaManager.UserSDK/MediaManager.cs:64]: Can we just do this:
                ServicePointManager.SecurityProtocol = (SecurityProtocolType)3072; // SecurityProtocolType.Tls12;
and avoid creating extensions?

File: Granicus.MediaManager.UserSDK/MediaManager.cs [hunk 2]
```diff
@@ -76,6 +77,7 @@ public MediaManager()
         /// <param name="Password">The password (e.g. secret) that matches the given username record.</param>
         public MediaManager(string Server, string Username, string Password)
         {
+            System.Net.ServicePointManager.SecurityProtocol = SecurityProtocolTypeExtensions.Tls12;
```
Comment (umidt) [inline Granicus.MediaManager.UserSDK/MediaManager.cs:80]: ServicePointManager.SecurityProtocol = (SecurityProtocolType)3072; // SecurityProtocolType.Tls12;

File: Granicus.MediaManager.UserSDK/MediaManager.cs (hunk 1 above)
Comment (mikephillipsjr) [reply Granicus.MediaManager.UserSDK/MediaManager.cs:64]: I gave that version a try and the tests pass on Win10 but fail on Win7. Think the extension method gives us a little more flexibility OS wise.
Also, seems to the MS preferred method:
https://support.microsoft.com/en-us/help/3154520/support-for-tls-system-default-versions-included-in-the-net-framework

**Comment by danvau7** [comment]
Any way to bump to 1.3? 1.2 is minimum now. Sorry to ask, but this touches lots of government stuff.

**Pull request merged**